src/components/sidebar/Sidebar.tsx

```diff
@@ -31,26 +28,86 @@ export const Sidebar: React.FC = () => {
     }
   }, [selectedMenu]);
 
+  const handleMenuItemClick = (page: string) => {
+    setSelectedNavItem(page);
+    setSelectedMenu(false);
+  };
+
   return (
-    <aside className="page__menu menu" id="menu">
+    <aside className="menu" id="menu">
       <div className="container">
-        <div className="menu__bottom">
+        <div className="menu__content">
           <nav className="menu-nav menu__nav">
             <ul className="menu-nav__list">
               <li className="menu-nav_item">
-                <Link className={`menu-nav__link menu-nav__link ${selectedNavItem === 'Home' ? 'is-active' : ''}`} to="/">Home</Link>
+                <Link
+                  className={`menu-nav__link ${selectedNavItem === 'Home' ? 'is-active' : ''}`}
+                  to="/"
+                  onClick={() => handleMenuItemClick('Home')}
+                >
+                  Home
+                </Link>
```

Comment:
just a thought here: as you are implementing the same logic throughout this part of the code, wouldn't it be more concise creating a specific function to deal with this?